GoogleForCreators/web-stories-wp
# Settings: Fix Stories Archives setting reset

## Context



## Summary

Reset local state for selected page when change archive type. 

## Relevant Technical Choices



## To-do



## User-facing changes


https://user-images.githubusercontent.com/237508/152337322-78b88c94-9b66-432d-a532-b8974a93375c.mov

## Testing Instructions






This PR can be tested by following these steps:

1. Go to Dashboard 
2. Go to settings. 
3. Change archive type to custom. 
4. Search for a page. 
5. Select page. 
6. Change archive type to default / disabled. 
7. Change archive type to custom. 
8. See that search input is empty. 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #10191
